exportlegends.lua

```diff
@@ -746,6 +790,8 @@ function export_more_legends_xml()
                     file:write("\t\t<"..k..">"..df_enums.profession[v]:lower().."</"..k..">\n")
                 elseif df.history_event_change_creature_typest:is_instance(event) and (k == "old_race" or k == "new_race")  and v >= 0 then
                     file:write("\t\t<"..k..">"..escape_xml(df.global.world.raws.creatures.all[v].name[0]).."</"..k..">\n")
+                elseif tostring(v):find ("<") then
```

Comment:
This is a good check! I think it might be better to log to the console instead of writing this to the file, though - people might not notice if it's only written to the file. (Maybe once for each unique value of k?)